### DISCUSSION TITLE
v0.16.0

### DISCUSSION THREAD

**Discussion by 4gray** [4gray/iptvnator#433] [General]
<img width="945" alt="big-iptvnator-v016" src="https://github.com/user-attachments/assets/bb93c25c-cdbc-4437-bddf-9147f21ed613">

💬  Telegram channel: https://t.me/iptvnator
🚀  Support project: https://github.com/sponsors/4gray

## What's Changed
* Xtream: New layout and EPG View
* Xtream: Support youtube trailers
* PWA: Updates in mobile layout
* Stalker Portal: Category and search fix
* Stalker & Xtream: Check playlist details from portal view
* Stalker & Xtream: Show IMDB rating information on overview page
* Stalker & Xtream: show stream URL in built-in video-player dialog
* Language: Update Italian language by @bovirus and @Deltwin 
* Language: Add Turkish language support by @hasanbeder 
* Style: Material 3 update by @4gray 

## New Contributors
* @Deltwin made their first contribution in https://github.com/4gray/iptvnator/pull/358
* @hasanbeder made their first contribution in https://github.com/4gray/iptvnator/pull/382

**Full Changelog**: https://github.com/4gray/iptvnator/compare/v0.15.1...v0.16.0

<hr /><em>This discussion was created from the release <a href='https://github.com/4gray/iptvnator/releases/tag/v0.16.0'>v0.16.0</a>.</em>

**Comment by elkir0** [comment]
Thanx, 
I can't get the Mac ARM version to work, Tryed on 2 computer (M1 and M2) always same error: Damaged File error.
any idea?

**Reply by 4gray** [reply]
Yes, that happens because the app is not signed, try this please: https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/
